packages/react-components/src/icons/filter-icon.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Can you add this icon to the icon story please :) I think the one you added in the other PR may also be missing